src/app/components/game/follow-widget/follow-widget.ts

```diff
@@ -90,14 +102,16 @@ export default class AppGameFollowWidget extends Vue {
 			// Do this before attempting to follow.
 			// We don't want to wait till the follow is confirmed to show the dialog,
 			// and even if the follow fails it's not like we'll close it.
-			if (this.shouldShowFollow && !this.game.developer.is_following) {
+			if (this.shouldShowFollow && UserFollowSuggestion.canSuggest(this.game.developer.id)) {
```

Comment:
Shouldn't the canSuggest function call be put into the getter for `shouldShowFollow`? or is it because it won't trigger the change detection? If so, put a comment